src/components/account/UserButtons.vue

```diff
@@ -4,20 +4,40 @@
     data-qa="user buttons"
   >
     <client-only>
-      <b-button
-        class="icon-ic-add"
-        data-qa="add button"
-        :aria-label="$t('set.actions.addTo')"
-        @click="addToSet"
-      />
-      <b-button
-        :pressed="liked"
-        class="icon-heart"
-        data-qa="like button"
-        :aria-label="$t('actions.like')"
-        size="sm"
-        @click="toggleLiked"
-      />
+      <template
```

Comment:
Let's move the recommendation buttons and accompanying logic out into a new component, `src/components/recommendation/RecommendationButtons.vue`

Then, in ItemPreviewCard.vue:
```html
    <template v-slot:buttons>
      <RecommendationButtons
        v-if="recommendedItem"
        v-model="identifier"
      />
      <UserButtons
        v-else
        v-model="identifier"
      />
    </template>
```